x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,55 @@
+.appSearchResult {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSizeXS * 5;
+    border-right: $euiBorderThin;
+  }
+
+  &__contentInner {
+    display: grid;
+    grid-gap: $euiSizeS;
+    overflow: hidden;
+  }
+
+  &__fieldsetContainer {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__hiddenFieldsIndicator {
+    font-size: $euiFontSizeXS;
+    color: $euiColorDarkShade;
+    align-self: end;
+    padding-top: 0;
+  }
+
+  &__toggleExpandButton {
```

Comment:
I might also suggest renaming this to a more generic `__actionButton` - in anticipation for the upcoming Curation results view

<img width="1017" alt="Screen Shot 2020-12-07 at 11 13 36 AM" src="https://user-images.githubusercontent.com/549407/101394395-4e42cc00-387d-11eb-9f91-c135c96ad04c.png">

After inspecting the current ent-search CSS, I realize ent-search uses a separate `toggle-expand-button` class and an `extra-button` class, I'm not a huge fan of that and I definitely think we can DRY that out to just a single reusable CSS class.